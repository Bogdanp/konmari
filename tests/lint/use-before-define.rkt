#lang racket/base

(define (f x)
  (define y (g x))
  y)

(define (g x)
  (add 1 x))
