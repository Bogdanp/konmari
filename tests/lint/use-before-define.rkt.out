/use-before-define.rkt:3:10:warning:identifier 'f' is never used
